packages/twenty-front/src/modules/action-menu/actions/record-agnostic-actions/constants/RecordAgnosticActionsConfig.tsx

```diff
@@ -70,4 +70,23 @@ export const RECORD_AGNOSTIC_ACTIONS_CONFIG: Record<string, ActionConfig> = {
     hotKeys: ['@'],
     shouldBeRegistered: () => true,
   },
+  [RecordAgnosticActionsKeys.OPEN_AI_CHAT]: {
+    type: ActionType.Standard,
+    scope: ActionScope.Global,
+    key: RecordAgnosticActionsKeys.OPEN_AI_CHAT,
```

Comment:
I know it was in the Figma but I think it should at least be plural ("AI Chats"). And maybe "View Previous" rather than Open?